### PR TITLE
Experimenting with installation type explanations

### PR DESCRIPTION
**Pull request by maxwelldb**
@vikram-redhat 

I'm thinking about different ways to approach https://bugzilla.redhat.com/show_bug.cgi?id=1801980 here. 

Any thoughts?

### CODE REVIEW

File: installing/install_config/installation-types.adoc [hunk 1]
```diff
@@ -48,6 +63,17 @@ Not all installation options are currently available for all platforms, as shown
 |
 |
 
+|Restricted network
+|✓
```
Comment (maxwelldb) [inline installing/install_config/installation-types.adoc:67]: Could just switch the check with `ND` or something.

**Comment by vikram-redhat** [comment]
@maxwelldb good concise approach to solving this. 

I like Approach 1. It is clear with the links to the actual docs.

However, I would like the wider team to weigh in on this as well. I will add this to the agenda for the meeting on Monday.

**Comment by jboxman** [comment]
+1 checkmarks!

File: installing/install_config/installation-types.adoc (hunk 1 above)
Comment (maxwelldb) [reply installing/install_config/installation-types.adoc:67]: If any changes happen to this page, my inclination is to stick to vague tweaks to the text. I'm not sure if committing to anything else is all that maintainable.

**Comment by maxwelldb** [comment]
Closing this. Will resolve https://bugzilla.redhat.com/show_bug.cgi?id=1801980 with a clean PR.

**Pull request closed without merging**